runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.client.plugins.clanchat;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	name = "Clan Chat",
+	keyName = "clanchat",
+	description = "Configuration for the clan chat plugin"
+)
+public interface ClanChatConfig extends Config {
```

Comment:
Remember to keep all braces on their own lines, the build won't pass checkstyle otherwise.